main/src/test/java/org/mobilitydata/gtfsvalidator/validator/StopLatLongValidatorTest.java

```diff
@@ -0,0 +1,153 @@
+package org.mobilitydata.gtfsvalidator.validator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.util.List;
+import javax.annotation.Nullable;
+import org.junit.Test;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.ValidationNotice;
+import org.mobilitydata.gtfsvalidator.table.GtfsLocationType;
+import org.mobilitydata.gtfsvalidator.table.GtfsStop;
+
+public class StopLatLongValidatorTest {
+  private static GtfsStop createStop(
```

Comment:
In general, I'd prefer to inline this method and drop calls to setStopLat() and setStopLon() where stop location data is missing.  In tests, I tend to favor readability in the unit test itself over general DRY conventions for regular code.